README.md

```diff
@@ -1,2 +1,32 @@
-# SSPINN
-Spectral Structure Prediction In Neural Networks
+![SSPINN Logo](./docs/images/SSPINN_Logo.png)
+
+[![Build Status](https://travis-ci.org/awild82/SSPINN.svg?branch=master)](https://travis-ci.org/awild82/SSPINN)[![Coverage Status](https://coveralls.io/repos/github/awild82/SSPINN/badge.svg?branch=master)](https://coveralls.io/github/awild82/SSPINN?branch=master)
+SSPINN is a software package meant to predict the structure of organic molecules based on experimentally obtained spectral data. Based on a user's input data, the SSPINN neural network will generate a connectivity matrix which predicts the bonding behavior between atoms in the user's molecule. This connectivity matrix will then be used to generate a graphical depiction of the predicted molecular structure. 
+
+## Software Requirements
+* [Python3](url)
+* [Keras](url)
+* [Networkx](url)
+    
+## Installation Instructions
+1. Clone git repository
+    *`git clone https://github.com/awild82/SSPINN.git`
+2. Install SSPINN by running setup file
+    *`./setup.sh`
```

Comment:
There is no `setup.sh`? Also, it is likely that users with different interpreter locations (i.e. if there is no `/bin/python`) can't do this. Moreover, they may not have administrative privileges. The general installation should be `python setup.py install --user`